pkgs/by-name/ba/bartender/package.nix

```
@@ -32,17 +39,37 @@ stdenvNoCC.mkDerivation (finalAttrs: {
     runHook postInstall
   '';
 
+  passthru.updateScript = lib.getExe (writeShellApplication {
+    name = "bartender-update-script";
+    runtimeInputs = [
+      curl
+      xmlstarlet
+      common-updater-scripts
+    ];
+    text = ''
+      version_major="${lib.versions.major finalAttrs.version}"
+      url="https://www.macbartender.com/B2/updates/AppcastB$version_major.xml"
+      version=$(curl -s "$url" | xmlstarlet sel -t -v '(//item)[last()]/sparkle:shortVersionString' -n)
+      update-source-version bartender "$version" --file=./pkgs/by-name/ba/bartender/package.nix
```

```suggestion
      update-source-version bartender "$version"
```
`update-source-version` *should* also be able to find this on it's own. Correct me if I'm wrong though :p

---

Ahh, good catch! I remember at one point it wasn't possible; it seems like it's not the case anymore